app/dashboard/sync/helpers.py

```diff
@@ -26,7 +26,7 @@ def record_payout_activity(fulfillment):
     kwargs['profile'] = fulfillment.funder_profile
     kwargs['metadata']['from'] = fulfillment.funder_profile.handle
     kwargs['metadata']['to'] = fulfillment.profile.handle
-    kwargs['metadata']['payout_amount'] = fulfillment.payout_amount
+    kwargs['metadata']['payout_amount'] = str(fulfillment.payout_amount)
```

Comment:
why does this change ? does this affect the round function that is used in frontend

Reply:
there's always been an error like `object of type Decimal is not JSON serializable` while running sync_pending_fulfillments

Reply:
Did we check to make sure that does this not have any side affects ? @chibie